src/rgw/rgw_common.cc

```diff
@@ -708,6 +708,9 @@ void RGWHTTPArgs::get_bool(const char *name, bool *val, bool def_val)
 
 bool verify_bucket_permission(struct req_state *s, int perm)
 {
+  if ((s->auth_method).get_acl_main_override())
+    return true;
```

Comment:
Add log line that ACL decision will be overridden. Use DSS INFO tag so that we know its ours.